Client/src/Service/Mixins/SearchReportsService.ts

```diff
@@ -68,8 +68,15 @@ export default (base: ServiceBase) => {
     const question = await base.findQuestion(answerSpec.searchName);
     const recordClass = await base.findRecordClass(question.outputRecordClassName);
     let url = base.getStandardSearchReportEndpoint(recordClass.urlSegment, question.urlSegment);
-    let searchConfig: SearchConfig = answerSpec.searchConfig
-    let body: StandardSearchReportRequest = { searchConfig, reportConfig };
+    let searchConfig: SearchConfig = answerSpec.searchConfig;
+    const reportConfigWithResponseBufferingSet: StandardReportConfig = {
+      ...reportConfig,
+      bufferEntireResponse: reportConfig.bufferEntireResponse ?? false,

```

Comment:
I was thinking that this should default to `true`. What do you think?

Reply:
Ah, I see. I got terribly confused. That makes more sense now. I will amend this PR!

Reply:
> I think we want to apply it to any method making a fetch request to the standard json reporter endpoint. The only place where we definitely want it to be false is in the `downloadAnswer` method. Does this sound right to you?

So would this mean making a PR `web-app-preferred-organisms` to add the `bufferEntireResponse` flag here: https://github.com/VEuPathDB/web-app-preferred-organisms/blob/main/src/lib/utils/organismMetadata.ts#L43 or here: https://github.com/VEuPathDB/web-app-preferred-organisms/blob/main/src/lib/utils/datasetMetadata.ts#L56?